options/locale/locale_en-US.ini

```diff
@@ -837,7 +837,7 @@ settings.search_user_placeholder = Search user...
 settings.org_not_allowed_to_be_collaborator = Organization is not allowed to be added as a collaborator.
 settings.user_is_org_member = User is organization member who cannot be added as a collaborator.
 settings.add_webhook = Add Webhook
-settings.hooks_desc = Webhooks are much like basic HTTP POST event triggers. Whenever something occurs in Gitea, we will handle the notification to the target host you specify. Learn more in this <a target="_blank" rel="noopener" href="%s">Webhooks Guide</a>.
+settings.hooks_desc = Webhooks are much like basic HTTP POST event triggers. Whenever something occurs in Gitea, we will send a notification to the target host. Learn more in the <a target="_blank" rel="noopener" href="%s">webhooks Guide</a>.
```

Comment:
If you're going to decapitalize "webhooks" (in "webhooks guide" at the end of the line), I would also decapitalize "guide"